binder/views.py

```diff
@@ -35,6 +35,25 @@
 from .json import JsonResponse, jsonloads
 
 
+def get_joins_from_queryset(queryset):
```

Comment:
Maybe add comments here on how it works/ and maybe a small docstring what it returns?